web/src/app/history/task-query/task-query.component.ts

```diff
@@ -184,13 +183,8 @@ export class TaskQueryComponent implements OnInit {
       false
     ).subscribe(taskQueryResource => {
       this.requestInProgressService.setRequestInProgress(false);
-      if (!taskQueryResource.taskHistoryEvents) {
```

Comment:
if case not needed??

Reply:
I inserted the if case by inserting ' = taskQueryResource.taskHistoryEvents ? ...'

So we get the same behavior in two lines and without an empty return statement